.github/workflows/test.yml

```diff
@@ -89,10 +89,9 @@ jobs:
 
     steps:
       - name: Install Python
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v5
         with:
-          python-version: '3.8'
-          architecture: 'x64'
+          python-version: '3.10'
```

Comment:
hm, should this be `${{ matrix.python-version }}`?

Reply:
Right, I don't recall we had `3.8` specified before.